lms/djangoapps/support/tests/test_views.py

```diff
@@ -65,11 +66,15 @@ class SupportViewManageUserTests(SupportViewTestCase):
     Base class for support view tests.
     """
 
+    ZENDESK_URL = 'http://zendesk.example.com/'
+    UNDEFINED_ZENDESK_URL = ''
+
     def setUp(self):
         """Make the user support staff"""
         super().setUp()
         SupportStaffRole().add_users(self.user)
 
+    @override_settings(ZENDESK_URL=ZENDESK_URL)
```

Comment:
why is this override needed? Is something wrong/misconfigured for this existing test?

Reply:
As by default ZENDEK_URL is an empty string, I have to override it to a non-empty one so it does not redirect